src/utils/hooks/use-field-state.ts

```diff
@@ -0,0 +1,22 @@
+import React, { Dispatch, SetStateAction, useCallback, useState } from 'react';
+
+export interface FieldState {
+    value: string;
+    onChange: React.ChangeEventHandler;
+    setValue: Dispatch<SetStateAction<string>>;
+    isPristine(trim: boolean): boolean;
```

Comment:
Hva betyr pristine..? :-P Kanskje et enklere ord for oss stakkars plebs? ;-)

Reply:
Det er i bunn og grunn det motsatte av `dirty`, og sjekker om `value === initialValue`.
Konvensjon brukt av redux-forms et.al,